lib/system/database/definitions/mysql.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'system/database/mysql'
```

Comment:
What? :confused: So `lib/system/database/definitions/mysql.rb` needs `require 'system/database/mysql'`, but `ib/system/database/definitions/oracle.rb` and `ib/system/database/definitions/postgres.rb` don't need `system/database/oracle` and `system/database/postgres` ? why?

Reply:
I was testing only where to add the system database 😄

Reply:
this is a total wip PR 😂